java/lp-service-spring-boot-3.gradle

```diff
@@ -29,7 +29,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath "org.springframework.boot:spring-boot-gradle-plugin:3.2.2"
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:3.3.5"
```

Comment:
Should we maybe use $springBootVersion?

Reply:
We could, I wasn't sure  as it was hardcoded. Let's set springBootVersion and handle if any issues arise